src/d1to2fix/main.d

```diff
@@ -150,7 +150,15 @@ int main ( string[] args )
     // figure it out
 
     import d1to2fix.symbolsearch;
-    config.importPaths ~= "/usr/include/d2/dmd-transitional/";
+    import std.file : exists;
+
+    if (exists("/usr/include/d2/dmd-transitional"))
+        config.importPaths ~= "/usr/include/d2/dmd-transitional/";
+    else if (exists("/usr/include/dmd/druntime/import"))
+        config.importPaths ~= "/usr/include/dmd/druntime/import/";
+    else if (exists("/usr/include/dlang/dmd"))
+        config.importPaths ~= "/usr/include/dlang/dmd/";
```

Comment:
If anyone wonders - the fix is not for d1to2fix itself but to be able to run its tests on systems that don't have dmd-transitional available, as we don't have public package for it.